script.js

```diff
@@ -1040,6 +1041,7 @@
                         font-size: 14px;
                         padding: 16px;
                         line-height: 20px;
+                        border-radius: 20px;
```

Comment:
![image](https://user-images.githubusercontent.com/20955511/224542024-f83b0665-9c08-40b3-a48f-2ef77ca3cb37.png)
![image](https://user-images.githubusercontent.com/20955511/224542114-ac1ebf0e-fd99-41a6-9069-c09180c740f7.png)

```suggestion
                        border-radius: 7px;
```